core-services/prow/02_config/_config.yaml

```diff
@@ -2368,6 +2365,17 @@ tide:
     - do-not-merge/invalid-owners-file
     repos:
     - operator-framework/operator-lib
+  - labels:
+    - lgtm
+    - approved
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
+    - bugzilla/invalid-bug
+    repos:
+    - openshift/cluster-baremetal-operator
```

Comment:
Please do not invent new query, but add the repo to one of the queries that already exist. Looks like the one on L2206 or L2259 should the suitable (looks like they have same criteria).